DataFormats/Scouting/test/TestRun3ScoutingFormats.sh

```diff
@@ -46,6 +50,11 @@ cmsRun ${LOCAL_TEST_DIR}/test_readRun3Scouting_cfg.py || die "Failure using test
 # for 2023 data, and the scouting data formats and ROOT were
 # not updated after 13_0_3.
 
+file=testRun3Scouting_v3_v7_v3_v4_v5_v3_v6_v3_v4_CMSSW_14_0_0_pre3.root
+inputfile=$(edmFileInPath DataFormats/Scouting/data/$file) || die "Failure edmFileInPath DataFormats/Scouting/data/$file" $?
+argsPassedToPython="--inputFile $inputfile --outputFileName testRun3Scouting2_CMSSW_14_0_0_pre3.root --electronVersion 7 --photonVersion 6 --vertexVersion 4"
+cmsRun ${LOCAL_TEST_DIR}/test_readRun3Scouting_cfg.py $argsPassedToPython || die "Failed to read old file $file" $?
+
```

Comment:
Thanks @mmusich! Could you either move this test to the last, or reorder the 12_4_0 and 13_0_3 tests, so that the tests would be in "time order"? Otherwise the PRs look good.

Reply:
done at [98d0560](https://github.com/cms-sw/cmssw/pull/43856/commits/98d056017f541a7420b08033f5a5aea7499c63f5)